.github/workflows/release-please.yml

```diff
@@ -13,7 +13,10 @@ jobs:
         id: release
         with:
           release-type: node
-      - uses: actions/checkout@v2
+
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.after }}
```

Comment:
optional nit: It would be helpful if you put some reason why we need to add this additionally instead of the hyperlink.

Reply:
The default github.ref is refs/heads/main, which I believe creates a race.  If something is pushed after the release PR but before this workflow runs, the wrong version could be checked out and released.